content/_index/coffeeeditoroverview/index.md

```diff
@@ -3,11 +3,12 @@ fragment = "content"
 weight = 299    
 background = "dark"
 
-title = "Coffee Editor Overview"
+title = "Coffee Editor NG"
 #subtitle= "Column based items with icons"
 #title_align = "center" # Default is center, can be left, right or center
 +++
 <p style='text-align: center;'>
-An example web-based modeling tool using EMF.cloud technologies used for demonstration and documentation.
+The <em>Coffee Editor NG</em> is a comprehensive example modeling tool based on EMF.cloud technologies and can also act as an architecture blueprint for your custom modeling tool.
+This full-featured example tool is written entirely in Typescript and includes a central model hub serving a sample modeling language to a variety of editors, such as a diagram editor, a form-based editor, as well as a textual DSL editor.
 </p>
-<img src="images/coffeeeditordemo.gif" alt="Coffee Editor Overview" style="display: block; margin: auto;"/>
\ No newline at end of file
+<img src="images/coffeeeditordemo.gif" alt="Coffee Editor Overview" style="display: block; margin: auto;"/>
```

Comment:
Can we already provide an updated GIF or preferably a video that shows the NG. If possible in dark mode?

Reply:
it looks the same ;-).